sdk/Lusid.Sdk.Tests/tutorials/Instruments/EquityOptionExamples.cs

```diff
@@ -135,5 +136,96 @@ public void EquityOptionPortfolioCashFlowsExample(ModelSelection.ModelEnum model
             var equityOption = InstrumentExamples.CreateExampleEquityOption();
             CallLusidGetPortfolioCashFlowsEndpoint(equityOption, model);
         }
+        
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney)]
+        public void LifeCycleManagementForCashSettledEquityOption(ModelSelection.ModelEnum model)
```

Comment:
Does this test have a feature tag that you can add to it? Consider adding method comments since this test is fairly complex

Reply:
I have no idea where to find it. However, I think Sam is doing them shortly. 
Have added comment for the test, thanks.